docs/modules/publisher-dmmigration.md

```diff
@@ -0,0 +1,141 @@
+| -------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------ |
+| **MethodName**                   | **Request**                                                  | **Response**                                                 | **in 2.8.2** |
+| **PublishNodes**                 | EndpointUrl, List<OpcNodes>,  UseSecurity, UserName, Password | Status, List<StatusResponse>                                 | Yes                        |
+| **UnpublishNodes**               | EndpointUrl, List<OpcNodes>                                  | Status, List<StatusResponse>                                 | Yes                        |
+| **UnpublishAllNodes**            | EndpointUrl                                                  | Status, List<StatusResponse>                                 | Yes                       |
+| **GetConfiguredEndpoints**       | -                                                            | List<EndpointUrl>                                            | Yes                        |
+| **GetConfiguredNodesOnEndpoint** | EndpointUrl                                                  | EndpointUrl, List< OpcNodeOnEndpointModel >    where OpcNodeOnEndpointModel contains:    Id ExpandedNodeId OpcSamplingInterval OpcPublishingInterval DisplayName HeartbeatInterval SkipFirst | Yes                       |
+| **GetDiagnosticInfo**            | -                                                            | DiagnosticInfoMethodResponseModel                            | Yes                        |
+| **GetDiagnosticLog**             | -                                                            | MissedMessageCount, LogMessageCount, List<Log>               | No                      |
+| **GetDiagnosticStartupLog**      | -                                                            | MissedMessageCount, LogMessageCount, List<Log>               | No                      |
+| **ExitApplication**              | SecondsTillExit (optional)                                   | StatusCode, List<StatusResponse>                             | No                      |
+| **GetInfo**                      | -                                                            | GetInfoMethodResponseModel >>    VersionMajor  VersionMinor  VersionPatch  SemanticVersion  InformationalVersion OS  OSArchitecture  FrameworkDescription | No                      |
+
+A [sample configuration application](https://github.com/Azure-Samples/iot-edge-opc-publisher-nodeconfiguration) as well as an [sample application for reading diagnostic information](https://github.com/Azure-Samples/iot-edge-opc-publisher-diagnostics) are provided from OPC Publisher open-source, leveraging this interface.
+
+_Hint: The samples applications are not actively maintained and may be outdated._
+
+## Direct Methods of version 2.8.2
```

Comment:
I think the sections below belong in the previous doc, where direct methods are described. I mean terminology,  payload schema, and so on. I think this information is more relevant for the new users of 2.8.2, that are not interested in migration steps. 

 We could add references to this information in the migration doc,

Reply:
Makes sense. Since I added request and response for 2.5.x as well in migration doc, I will move these terms in another doc.